python/fastquant/test.py

```diff
@@ -0,0 +1,13 @@
+from fastquant import get_yahoo_data, get_bt_news_sentiment, get_twitter_sentiment, tweepy_api
+
+
```

Comment:
This test module should be moved to:
https://github.com/enzoampil/fastquant/tree/master/python/tests as `test_twitter.py`
following the pytest format:
```python
from fastquant import get_yahoo_data, get_bt_news_sentiment, get_twitter_sentiment, tweepy_api
import pandas as pd

consumer_key = ''
consumer_secret = ''
access_token = ''
access_secret = ''

account_list = ['2TradeAsia', 'colfinancial']

def test_twitter():
    api = tweepy_api(consumer_key, consumer_secret, access_token, access_secret)
    sentiment = get_twitter_sentiment('$ALI', api, '2020-06-14', account_list)
    assert isinstance(sentiment, pd.DataFrame)
```

Reply:
Got it, will do the necessary edits. Thanks @jpdeleon